inspector/src/components/actionTabs/tabs/propertyGrids/animations/curveEditor/graph/rangeFrameBarComponent.tsx

```diff
@@ -113,9 +113,9 @@ export class RangeFrameBarComponent extends React.Component<IRangeFrameBarCompon
         const to = this.props.context.toKey;
 
         let range = to - from;
-        let stepCounts = Math.min(20, to - from);
-        let offset = (range / stepCounts) | 0;
         let convertRatio = range / this._viewWidth;
+        let dist = 25; // x distance between consecutive ticks
+        let offset = Math.max(Math.floor(dist * convertRatio), 1);

```

Comment:
could be consts.

Also, can the default values be predefined as constants somewhere?

Reply:
Updated!